src/elements/kw-share-detail/kw-share-detail.html

```diff
@@ -1,101 +1,241 @@
+				height: 32px;
+				color: white;
+				border: none;
+			}
+			
+			.remixbutton {
+				width: 79px;
+			}
+			
+			.bottom .line {
+				height: 1px;
+				width: 100%;
+				display: block;
+				background-color: rgba(99, 107, 126, .24);
+			}
+			
+			div[tombstone].bottom .likes,
+			div[tombstone].bottom .comments,
+			div[tombstone].bottom .shares {
+				width: 80px;
+				height: 20px;
+				background: rgba(0, 0, 0, 0.01);
+				color: transparent;
+			}
+			
+			.orangebutton {
+				background-color: #FF8413;
+				border-radius: 5px;
+				width: 79px;
+				height: 32px;
+				color: white;
+				border: none;
+			}
+			
+			.remixbutton {
+				width: 79px;
+			}
+			.playercontrol {
+				@apply --layout-horizontal;
+				@apply(--layout-center);
+				width: 100%;
+				margin: 9px;
+			}
+			
+			.playercontrol .pause {
+				@apply(--layout-justified);
+			}
+			
+			.playercontrol .replay {
+				@apply(--layout-justified);
+			}
+			
+			.playercontrol .lineV {
+				@apply(--layout-justified);
+				height: 20px;
+				width: 1px;
+				display: block;
+				background-color: rgba(99, 107, 126, .24);
+				margin-right: 16px;
+				margin-left: 16px;
+			}
+			
+			.playercontrol .fullscreen {
+				@apply --layout-end-justified;
+				padding-right: 9px;
+			}
+			
+			.space {
+				@apply(--layout-flex);
+			}
+		</style>
+		<kw-share-shell id$="[[share.slug]]" share="[[share]]">
+			<div slot="share-detail">
+				<div id="share" class="detail">
+					<iron-image src="[[share.cover_url]]" sizing="cover"></iron-image>
+				</div>
+				<div class="bottom" tombstone$="[[!share]]">
+					<div class="playercontrol">
+						<div class="pause">
+							<span><img src="../../assets/icons/shares/pause.svg"></img></span>
+						</div>
+						<div class="lineV"></div>
+						<div class="replay">
+							<span><img src="../../assets/icons/shares/replay.svg"> </img></span>
+						</div>
+						<div class="space"></div>
+						<div class="fullscreen">
+							<span><img src="../../assets/icons/shares/fullscreen.svg"> </img></span>
+						</div>
+					</div>
+					<div class="line"></div>
+					<div class="content">
+						<div class="description">
+							<h3>[[share.title]]</h3>
+							<p>[[share.description]]</p>
+						</div>
+						<div class="actions">
+							<button type="submit" class="orangebutton remixbutton" onclick="">remix <img src="../../assets/icons/shares/remix.svg" /></button>
+						</div>
+					</div>
+					<div class="line"></div>
+					<div class="stats">
+						<div class="likes">
+							<span><img src="../../assets/icons/shares/like.svg"> </img>[[share.likes.length]] likes</span>
+						</div>
+						<div class="comments">
+							<span><img src="../../assets/icons/shares/comment_b.svg"> </img>[[share.comments_count]] comments</span>
+						</div>
+						<div class="shares">
+							<span><img src="../../assets/icons/shares/share.svg"> </img>share</span>
+						</div>
+					</div>
+				</div>
+				<kw-view-comment></kw-view-comment>
```

Comment:
The pattern `kw-view` is used to specify actual views of the app. Please find another, more fitting name